app/lib/rocketchat.js

```diff
@@ -540,11 +540,6 @@ const RocketChat = {
 			this.oldPromise('cancel');
 		}
 
-		if (searchText === '') {
```

Comment:
You can't remove this without add another logic to `delete` the `oldPromise` when search was finished.

Reply:
Ok, I'll look into it and will see what is required